py-polars/polars/expr/binary.py

```diff
@@ -289,3 +296,42 @@ def size(self, unit: SizeUnit = "b") -> Expr:
         sz = wrap_expr(self._pyexpr.bin_size_bytes())
         sz = scale_bytes(sz, unit)
         return sz
+
+
+
+    def from_buffer(
+        self, dtype: PolarsDataType | type[Any], endianness: Endianness = "little"
```

Comment:
Let's make this required keyword arguments:

`self, * , dtype: PolarsDataType | type[Any], endianness: Endianness = "little"`

Reply:
fixed.